### PR TITLE
Move character method from battle

### PR DESCRIPTION
**Pull request by BertovDev**
Closes 

## Motivation

Remove handle character methods from battle to customCharacter

## Summary of changes

Everything should work as always

## How has this been tested?

Describe the steps you performed to test this PR.

## Test additions / changes

List tests added/updated.

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

**Review by AngieDutra — DISMISSED** [review]
Nice ✨

**Review by AngieDutra — APPROVED** [review]
Tested it doing a match with h4ck and muflus and worked great!

File: client/Assets/Scripts/CustomCharacter.cs [hunk 1]
```diff
@@ -18,4 +19,60 @@ protected override void Initialization()
             this.characterBase.gameObject.AddComponent<AudioSource>();
         }
     }
+
+    public void rotatePlayer(GameObject player, Direction direction)
```
Comment (manucamejo) [inline client/Assets/Scripts/CustomCharacter.cs:23]: Can you use Uppercase here? I think it's a good practice in C#

File: client/Assets/Scripts/CustomCharacter.cs [hunk 2]
```diff
@@ -18,4 +19,60 @@ protected override void Initialization()
...
+        this.CharacterModel.SetActive(false);
+        this.ConditionState.ChangeState(CharacterStates.CharacterConditions.Dead);
+        this.characterBase.Hitbox.SetActive(false);
+        DestroySkillsClone();
+        this.GetComponentInChildren<CharacterBase>().OrientationIndicator.SetActive(false);
+        //Currently unused code
+        // if (GameServerConnectionManager.Instance.playerId == ulong.Parse(playerCharacter.PlayerID))
+        // {
+        //     CustomGUIManager.DisplayZoneDamageFeedback(false);
+        // }
```
Comment (agustinesco) [inline client/Assets/Scripts/CustomCharacter.cs:46]: Let's remove this

**Review by agustinesco — APPROVED** [review]
LGTM!

**Review by manucamejo — APPROVED** [review]

**Pull request merged**